.github/workflows/deploy-draft.yml

```diff
@@ -0,0 +1,86 @@
+name: Deploy branch draft on Netlify
+on:
+  pull_request:
+    types: [labeled, opened, synchronize]
+    
+jobs:
+  build-deploy:
+    if: |
+      (github.event.action == 'labeled' && github.event.label.name == 'deploy') ||
+      (github.event.action != 'labeled' && contains(github.event.pull_request.labels.*.name, 'deploy'))
+    runs-on: ubuntu-latest
+    steps:
+    - name: inject slug/short variables
+      uses: rlespinasse/github-slug-action@v3.x
+
+    - name: set STAGE variable in environment for next steps
+      run: echo "STAGE=pr-${{ github.event.number }}-${{ env.GITHUB_HEAD_REF_SLUG }}" >> $GITHUB_ENV
+
+    - name: create a github deployment
+      uses: bobheadxi/deployments@v0.5.2
+      id: deployment
+      with:
+        step: start
+        token: ${{ secrets.GITHUB_TOKEN }}
+        env: ${{ env.STAGE }}
+        ref: ${{ github.head_ref }}
+        
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 # fetch whole repo so git-restore-mtime can work
+
+    - name: Adjust file watchers limit
+      run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
+
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '14.x'
+    - name: Install yarn
+      run: sudo npm -g install yarn
+    - name: Yarn install
+      run: yarn install --immutable
```

Comment:
what do you think about having Yarn installed in the package.json as a dev dependency and then just use it here  `npx yarn install --immutable`?